tests/e2e/.mocharc.js

```diff
@@ -10,22 +10,22 @@ module.exports = {
   exit: true,
   require: 'node_modules/jet/platform/node',
   spec: [
-    '../packages/app/e2e/**/*.e2e.js',
-    '../packages/app-check/e2e/**/*.e2e.js',
-    '../packages/app-distribution/e2e/**/*.e2e.js',
-    '../packages/analytics/e2e/**/*.e2e.js',
-    '../packages/auth/e2e/**/*.e2e.js',
+    // '../packages/app/e2e/**/*.e2e.js',
+    // '../packages/app-check/e2e/**/*.e2e.js',
```

Comment:
I see these changes were reverted in a following commit - can just rebase them out of the commit chain completely with a `rebase -i` (probably `rebase -i HEAD~1` then mark the revert as a "squash", edit the commit notes in the next screen and poof, they weren't there to begin with